generators/app/templates/scripts/_terraform.sh.ejs

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
I can't comment on unchanged lines, but I think `terraform init` should be changed to:

```shell
    "init")
    echo $root/terraform
    cd $root/terraform
    terraform init -upgrade
    ;;
```

and to drop the `args`

Reply:
uh, why do we need to echo the path?

Reply:
updated to remove args and add upgrade flag